recipe/meta.yaml

```diff
@@ -22,46 +22,45 @@ requirements:
     - bzip2 1.0.*
     - cairo 1.14.*  # [linux]
     - curl >=7.54.1,<8
-    - freetype 2.7  # [linux]
+    - freetype 2.8.*  # [linux]
     - g2clib 1.6.*
-    - gsl  2.2.*
+    - gsl 2.2.*
     - hdf4
-    # FIXME: We cannot use hdf5 1.10.1 b/c of hdfeos5 is failing with that version.
-    - hdf5 1.8.18|1.8.18.*
+    - hdf5 1.10.1
     - hdfeos2
-    - hdfeos5
+    # FIXME: We cannot use hdfeos5 b/c it won't compiled with hdf5 1.10.1
+    # - hdfeos5
```

Comment:
@khallock is there a way to disable `hdfeos5`? Downgrading to `hdf5 1.8.18` just for that dependency will be quire time consuming and beyond my allocated time as a volunteer for `conda-forge`.

Reply:
@ocefpaf, I just pushed commit f5f8697 to your branch, which should prevent `hdfeos5` support from being built.

Reply:
Thanks! Let's hope that is the only hiccup preventing the build.